hooks/pre/10-reset_data.rb

```diff
@@ -52,7 +52,7 @@ def empty_db_in_postgresql(db)
 end
 
 def reset_candlepin
-  execute('rm -f /var/lib/candlepin/{cpdb_done,cpinit_done}')
+  execute('rm -f /var/lib/candlepin/.puppet-candlepin*')
```

Comment:
Let's be slightly stricter:
```suggestion
  execute('rm -f /var/lib/candlepin/.puppet-candlepin-cpdb-*')
```

Maybe we need to store more state in the future

Reply:
Fair.